app/assets/stylesheets/administrate/components/_pagination.scss

```diff
@@ -1,5 +1,6 @@
 .pagination {
-  margin: $base-spacing 0;
+  @include margin($base-spacing null ($base-spacing * 4));
```

Comment:
`$base-spacing * 4` should be written without spaces around the operator: `$base-spacing*4`